packages/driver/src/cy/commands/local_storage.js

```diff
@@ -58,5 +58,14 @@ module.exports = (Commands, Cypress, cy, state) => {
       // return the remote local storage object
       return remote
     },
+
+    // TODO: add these user-facing APIs
+    // getLocalStorage (options) {
+    //   return Cypress.session.getLocalStorage(options)
+    // },
+
+    // setLocalStorage (options) {
+    //   return Cypress.session.setLocalStorage(options)
+    // },
```

Comment:
Can we move this into an issue if it's something we'd like to add, then delete these lines from this PR? I feel like this is going to stay permanently commented out otherwise.

Reply:
85d85eaeeb